docker/dev/nginx.conf

```diff
@@ -1,21 +1,17 @@
 daemon off;
 user www-data;
-worker_processes 5;
+worker_processes auto;
 pid /run/nginx.pid;
 
 error_log /dev/stdout info;
 
 events {
     worker_connections 1024;
-    # multi_accept on;
+    multi_accept on;
 }
 
 http {
-    log_format main '$http_x_forwarded_for ($remote_addr) - $remote_user [$time_local] '
-        '"$request" $status $body_bytes_sent "$http_referer" '
-        '"$http_user_agent"' ;
-
-    access_log /dev/stdout main;
+    access_log off;
```

Comment:
Why turn off the access log for the dev configuration ? :thinking:

Reply:
@thomasbisignani, I try to optimize a bit nginx, because it isn't very fast on dev.
IMO this log is useless, have you ever had the need to see them?